health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UploadDataCustom.js

```
@@ -734,28 +734,22 @@ const UploadDataCustom = React.memo(({ formData, onSelect, ...props }) => {
-                    }}
-                    textStyles={{
-                      color: "#FFFFFF"
-                    }}
-                  />]
+                [<InfoButton
+                  infobuttontype="error"
+                  label={t("HCM_VIEW_ERROR")}
+                  onClick={() => setShowPreview(true)}
+                  size=""
+                  style={{}}
+                />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)